docs/view_components/create_scenario.rst

```diff
@@ -0,0 +1,54 @@
+..
+    one can then cross link to this view component by using
+    :ref:`create_scenario-label`
+
+Create Scenario
+---------------
+
+Create scenario view-component enables the user to create a new scenario object where the energy system can be implemented.
+
+Attributes
+^^^^^^^^^^
+
+**Combination of a text field and other GUI elements**
+
+    Properties:
+        * Allows user to navigate through the file system either graphically or through text inputs
+        * Allows user to name the scenario object and write it to a chosen location in the computer's file system
+
+    Requirements index: 1
+
+Actions
+^^^^^^^
+
+1. When a new scenario is created, it should appear in the the list of available scenarios within the project in the scenarios view
+
+Requirements
+^^^^^^^^^^^^
+
+1. A string containing the correct file path that leads to the repository that holds the scenario file(s)
```

Comment:
What is a "repository" here?

Reply:
A folder where the user will have to save the files. Without that I do not see a way to save the scenario objects

Reply:
If it is a "folder", then call it a "folder"